### PR TITLE
replace js script to htmx some refactoring

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

**Review by Taraslysak — APPROVED** [review]

File: app/views/main.py [hunk 1]
```diff
@@ -9,3 +9,9 @@
 @login_required
 def index():
     return render_template("index.html")
+
+
+@main_blueprint.route("/no-content")
+def no_content():
+    """htmx reauest"""
```
Comment (sa1am8) [inline app/views/main.py:16]: *htmx request

File: app/views/user.py [hunk 2]
```diff
@@ -22,16 +24,12 @@
 @login_required
 def get_all():
     q = request.args.get("q", type=str, default=None)
-    query = m.User.select().order_by(m.User.id)
-    count_query = sa.select(sa.func.count()).select_from(m.User)
+    where = m.User.is_deleted.is_(False)
```
Comment (sa1am8) [inline app/views/user.py:27]: change "where" to any other variable..
at least "stmt"

File: app/views/user.py [hunk 3]
```diff
@@ -44,31 +42,58 @@ def get_all():
     )
 
 
+@bp.route("/get-edit-form/<user_uuid>", methods=["GET"])
+@login_required
+def get_edit_form(user_uuid: str):
+    """htmx reauest"""
```
Comment (sa1am8) [inline app/views/user.py:48]: request

File: app/views/user.py [hunk 4]
```diff
@@ -44,31 +42,58 @@ def get_all():
...
+    user: m.User | None = db.session.scalar(m.User.select().where(m.User.uuid == user_uuid))
+    if not user or user.is_deleted:
+        log(log.ERROR, "User not found by id: [%s]", user_uuid)
+        return render_template("toast.html", category="danger", message="User not found"), 404
+    form = f.UserForm(
+        user_uuid=user.uuid,
```
Comment (sa1am8) [inline app/views/user.py:54]: maybe try to use f.UserForm(**user.__dict__)? its gonna be more flexible in future

**Review by sa1am8 — COMMENTED** [review]
Comments left

**Pull request merged**